README.md

```diff
@@ -991,7 +991,7 @@ There are five flags that you can use with this command:
 * `--latest`: Install the latest version of the module instead of specifying the version using the cli prompt.
 * `--force`: Overwrite a previously installed version of the module.
 * `--sha <commit_sha>`: Install the module at a specific commit from the `nf-core/modules` repository.
-* `--all`: Use this flag to change versions on all installed modules. Has the same effect as running `nf-core modules install --force` on all installed modules.
+* `--all`: Use this flag to change versions on all installed modules. Has the same effect as running `nf-core modules install --force --latest` on all installed modules. To change all modules to a specific version you can run `nf-core modules install --all --sha <commit sha>`.
```

Comment:
```suggestion
* `--all`: Use this flag to change versions on all installed modules. Has the same effect as running `nf-core modules install --force` on all installed modules. To change all modules to a specific version you can run `nf-core modules install --all --sha <commit sha>`.
```

Reply:
Sorry, I thought we were removing `--latest` altogether as an option?

Reply:
No, the `--latest` flag should be included. If we don't want to remove the cli prompt entirely?

Reply:
Ah, I see. So its one of those options we need to keep for the CLI prompt. Finally got there! 😅 Ok.